src/config/customToast/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { useMemo } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { FontSize } from '@themes';
+import Colors from '@themes/colors';
```

Comment:
why do we have two import statements from theme, either make it 

import FontSize from '@themes/fonts';
import Colors from '@themes/colors';

or 

import { FontSize, Colors } from '@themes';